packages/docs/stories/components/checkbox.stories.ts

```diff
@@ -0,0 +1,146 @@
+
+/* eslint-disable import/no-relative-packages */
+
+import '../../../components/src/components/checkbox/checkbox';
+import type { Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { userEvent } from '@storybook/testing-library';
+import { waitUntil } from '@open-wc/testing-helpers';
+import docsTokens from '../../../tokens/src/figma-tokens/_docs.json';
+import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../src/helpers/component.js';
+const { args, argTypes } = storybookDefaults('syn-checkbox');
+const { overrideArgs } = storybookHelpers('syn-checkbox');
+const { generateTemplate } = storybookTemplate('syn-checkbox');
+
+const generateStoryDescription = (attributeName: string = '') => {
+  const story = attributeName
+    ? (docsTokens?.components?.checkbox as Record<string, any>)?.[attributeName]?.description?.value ?? 'No Description'
+    : (docsTokens?.components?.checkbox as Record<string, any>)?.description?.value ?? 'No Description';
+  return {
+    story,
+  }
+};
+
+const meta: Meta = {
+  component: 'checkbox',
+  args: overrideArgs([
+    { name: 'default', type: 'slot', value: 'Checkbox' }
+  ]),
+  argTypes,
+  title: 'Components/syn-checkbox',
+  parameters: {
+    docs: {
+      description: generateStoryDescription(),
+    }
+  }
+};
+export default meta;
+
+type Story = StoryObj;
+
+export const Default = {
+  render: (args: any) => {
+    return generateTemplate({ args });
+  },
+  parameters: {
+    docs: {
+      description: generateStoryDescription(),
+    }
+  }
+} as Story;
+
+
+/**
+ * Use the checked attribute to activate the checkbox.
+ */
+export const Checked: Story = {
+  render: () => html`<syn-checkbox checked>Checked</syn-checkbox>`,
+};
+
+/**
+ * Use the indeterminate attribute to make the checkbox indeterminate.
+ */
+export const Indeterminate: Story = {
+  render: () => html`<syn-checkbox indeterminate>Indeterminate</syn-checkbox>`,
+};
+
+/**
+ * Use the disabled attribute to disable the checkbox.
+ */
+export const Disabled: Story = {
+  render: () => html`<syn-checkbox disabled>Disabled</syn-checkbox>`,
+};
+
+/**
+ * Use the size attribute to change a checkbox's size.
+ */
+export const Sizes: Story = {
+  render: () => html`<syn-checkbox size="small">Small</syn-checkbox>
+<br />
+<syn-checkbox size="medium">Medium</syn-checkbox>
+<br />
+<syn-checkbox size="large">Large</syn-checkbox>`,
+};
+
+/**
+ * Use the setCustomValidity() method to set a custom validation message. This will prevent the form from submitting and make the browser display the error message you provide. To clear the error, call this function with an empty string.
+ */
+export const CustomValidity: Story = {
+  play: async ({ canvasElement }) => {
+    try {
+      const button = canvasElement.querySelector('button');
+
+      await waitUntil(() => customElements.whenDefined('syn-checkbox'));
+
+      if (button) {
+        await userEvent.click(button);
+      }
```

Comment:
I think it's a ShadowDOM problem. I would either try:
- `button.click()` <img width="1062" alt="CleanShot 2023-11-22 at 11 08 14@2x" src="https://github.com/SickDesignSystem/synergy/assets/26542182/08b08c1e-8e41-4ab0-aa1a-5cbf1d5b14dc">
- or jump into the ShadowDOM to get the button inside and make `userEvent.click(shadowedButton)`

Reply:
I tried it before reverting to the native button solution, did not work. I think I had seen something like this as a shoelace issue, but did not find it yet.